animation/03-dropdown-menu/README.md

```diff
@@ -0,0 +1,18 @@
+# Dropdown Menu
+
+We've set up a dropdown menu in this exercise. Load up the page, you can see a single menu title, with a dropdown menu that will open upon clicking on the title. 
+
+Your task is to add animation to the dropdown menu so that it will have an effect of growing down. Check out the desired outcome below, and notice the _bounce_ illusion when the dropdown grows close to its final end state. 
```

Comment:
I'm not 100% sold on using the word _growing_.

How do you feel about _expanding_ rather than _growing_ and _expands_ rather than _grows_ ?

Happy to hear your thoughts on it.

Reply:
Yes that sounds better :) I will make some changes.